OracleUnifiedDirectory/README.md

```diff
@@ -0,0 +1,134 @@
+Oracle Unified Directory (OUD) on Docker
+========================================
+
+## Contents
+
+1. [Introduction](#introduction)
+2. [Prerequisites for OUD on Docker](#prerequisites-for-oud-on-docker)
+3. [Prerequisites for OUD on Kubernetes](#prerequisites-for-oud-on-kubernetes)
+4. [Installing the OUD Docker Image](#installing-the-oud-docker-image)
+5. [Running the OUD Docker Image in a Docker Container](#running-the-oud-docker-image-in-a-docker-container)
+6. [OUD Docker Container Configuration](#oud-docker-container-configuration)
+7. [OUD Kubernetes Configuration](#oud-kubernetes-configuration)
+
+## Introduction
+
+Oracle Unified Directory provides comprehensive Directory Solution for robust Identity Management.
+
+Oracle Unified Directory is an all-in-one directory solution with storage, proxy, synchronization and virtualization capabilities. While unifying the approach, it provides all the services required for high-performance Enterprise and carrier-grade environments. Oracle Unified Directory ensures scalability to billions of entries, ease of installation, elastic deployments, enterprise manageability and effective monitoring.
+
+This project offers Dockerfiles and scripts to build and configure an Oracle Unified Directory image based on 12cPS4 (12.2.1.4.0) release. Use this Docker Image to facilitate installation, configuration, and environment setup for DevOps users. 
+
+This Image refers to binaries for OUD Release 12.2.1.4.0 and it has the capability to create different types of OUD Instances (Directory Service, Proxy, Replication) on containers targeted for development and testing.
+
+***Image***: oracle/oud:12.2.1.4.0
+
+## Prerequisites for OUD on Docker
+
+The following prerequisites are applicable when using the OUD Docker image on Docker only:
+
+* A working installation of Docker 18.03 or later
+
+## Prerequisites for OUD on Kubernetes
+
+Refer to the [Prerequisites](https://oracle.github.io/fmw-kubernetes/oud/prerequisites) in the Oracle Unified Directory Kubernetes documentation.
+
+## Installing the OUD Docker Image
+
+An OUD Docker image can be created and/or made available for deployment in a Docker environment in the following ways:
+
+1. Download a pre-built OUD Docker image from [My Oracle Support](https://support.oracle.com) (Patch 31979398).  This image contains the latest bundle patch and one-off patches for Oracle Unified Directory 12.2.1.4.0.
+2. Build your own OUD Docker image using the WebLogic Image Tool. Oracle recommends using the Weblogic Image Tool to build your own OUD 12.2.1.4.0 image along with the latest Bundle Patch and any additional patches that you require. For more information, see [Building an OUD Docker Image using Image Tool](https://github.com/oracle/docker-images/OracleUnifiedDirectory/imagetool/12.2.1.4.0)
+
+## Running the OUD Docker Image in a Docker Container
```

Comment:
Remove all instances of the word "Docker" and "container" should be lowercase, i.e. "Running OUD in a container"